blog/2024-01-25/index.md

```diff
@@ -0,0 +1,60 @@
+
+## How We Measure Growth
+
+We have a [public dashboard](https://flathub.org/statistics) that shares some basic statistics, powered by an [open source script](https://github.com/flathub-infra/flathub-stats). We don't track individual metrics from end user machines; instead, we look at downloads of specific artifacts from the Flathub infrastructure itself. This gives us a count of installs and updates over time, downloads per country (based on the origin of requests), and some derivative data like downloads of apps in different categories.
+
+But how can we measure how many _active users_ we have?
+
+Since Flathub necessarily serves downloads of Flatpak _runtimes_ (common platforms on which apps are built), we can estimate active users fairly well. For example, when installing or updating many apps, Flatpak will automatically install the base [FreeDesktop SDK](https://freedesktop-sdk.gitlab.io/) runtime on which the KDE and GNOME runtimes (and most other apps) are built. We can look at the number of downloads of _updates_ to a recent release of this runtime to estimate how many active installs are out there getting Flatpaks from Flathub.
+
+This methodology reveals that over the past few months there have been over a million updates of each of the latest FreeDesktop SDK runtime releases, meaning we've passed the one million active user mark.
+
+:::note
+If anything, we believe this to be a conservative estimate as some users may be using apps that do not use the specific runtime measured.
+:::
+
+If you'd like to explore more of this data yourself, there's this [handy dashboard by Kristian Klausen](https://klausenbusk.github.io/flathub-stats/#ref=org.freedesktop.Platform/23.08&interval=infinity&downloadType=updates) ([source](https://github.com/klausenbusk/flathub-stats)), or you can always chat with us in [#flathub on Matrix](https://matrix.to/#/#flathub:matrix.org).
+
+## What's Driving Growth
+
+We attribute the growth of Flathub to a few developments over the years, and especially the last several months:
+
+1. **Popular app availability**: Since we started Flathub, we've seen apps like Firefox, Google Chrome, Discord, VLC, Spotify, Telegram, Microsoft Edge, Steam, OBS Studio, Zoom, Thunderbird, and _many_ more make their way into the store. These are many of the apps people want and expect, especially if they're coming to a Linux desktop for the first time. Each time another popular app hits Flathub, it makes Flathub that much more compelling to users.
+
+2. **Verified apps**: We've heard that some folks have held off installing specific apps or even using Flathub altogether because they didn't want a third-party maintainer redistributing an app they rely on; verifying apps solves this by assuring users their favorite app is actually coming from its developer. With an ever-increasing number of apps choosing to get verified—including both big names and many newer indie apps—the trust and adoption of Flathub increases.
```

Comment:
For someone who doesn't know how verification works this might not be super clear, I'd maybe include a few words about how verification works, or at least a link to a more detailed explanation.

Reply:
I linked to verification earlier, but yeah, could do here, too

```suggestion
2. **Verified apps**: We've heard that some folks have held off installing specific apps or even using Flathub altogether because they didn't want a third-party maintainer redistributing an app they rely on; [verifying apps](/docs/for-users/verification/) solves this by assuring users their favorite app is actually coming from its developer. With an ever-increasing number of apps choosing to get verified—including both big names and many newer indie apps—the trust and adoption of Flathub increases.
```